EIPS/eip-1789.md

```diff
@@ -33,12 +33,14 @@ We recognize that, while technically trivial, the real challenge in inflation-ba
 
 Two new constants are introduced: BENEFICIARY_ADDRESS, an Address, and DEVFUND_BLOCK_REWARD, an amount denominated in wei. Both are set to zero.
 
-Beginning with block ISTANBUL_BLOCK_HEIGHT, a coinbase transaction is added to each block minting DEVFUND_BLOCK_REWARD wei and adding it to the balance of BENEFICIARY_ADDRESS.
+Beginning with block ISTANBUL_BLOCK_HEIGHT, if DEVFUND_BLOCK_REWARD is greater than zero, a coinbase transaction is added to each block minting DEVFUND_BLOCK_REWARD wei and adding it to the balance of BENEFICIARY_ADDRESS.
 
 ## Rationale
 <!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
 
-Pending
+We believe that the technical design of this EIP is straightforward.
+
+Community consensus around this proposal will be important to measure.  Conversations around the administration, governance, and deployment of funds  are ongoing [here](https://github.com/ethereum/EIPs/issues/1789) and [here](https://github.com/gitcoinco/ERC-1789/issues).  A tennagraph poll is planned to measure community consensus for this initiative.
```

Comment:
I'm on the fence about whether to include language like this in the EIP. 1. EIPs are supposed to be purely technical (I think), and 2. Even if we want to discuss sentiment, it would be better to describe _results_ than point at other threads that are "ongoing."

Reply:
ok, i will make some edits.  anything else that should go in the 'rationale' section?  something that feels better than "Pending"

Reply:
The description of that section says:

> The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.

We don't have much to say about what motivated the technical design because it's trivial. We will have a lot to say on "evidence of consensus within the community" and "important objections or concerns raised during discussion" but I don't think this debate has really begun yet.